### PR TITLE
Add tests for SSO session failover.

### PR DESCRIPTION
**Pull request by krausvo1**
I had to configure haproxy to ignore `amlbcookie` in case the target AM instance is not up and try to redirect the request to the second AM instance (by default it just returns 503). I have also considered sending the last request directly to the second AM instance (by setting `amlbcookie`) instead and not worrying about haproxy, but the chosen approach feels more complete and appropriate for theses tests.

### CODE REVIEW

File: audit/.support.sh [hunk 1]
```diff
@@ -36,6 +36,6 @@ assert_audit_file_content() {
 }
 
 set_lb_cookie() {
-  local amlbcookie=$([ $TEST_INSTANCE_ID == 2 ] && echo "03" || echo "0$TEST_INSTANCE_ID")
+  local amlbcookie=$([ $TEST_INSTANCE_ID = 2 ] && echo "03" || echo "0$TEST_INSTANCE_ID")
```
Comment (pavelhoral) [inline audit/.support.sh]: This contains incorrect logic that assumes cookie value can be mapped to instance ID.

Comment (krausvo1) [reply audit/.support.sh]: I have used associative array of AM instances and their respective LB cookie values instead.

**Review by pavelhoral — APPROVED** [review]
LGTM

**Review by pavelhoral — APPROVED** [review]
LGTM

**Pull request merged**